book/src/api-vc-endpoints.md

```diff
@@ -816,3 +817,48 @@ logs emitted are INFO level or higher.
   }
 }
 ```
+
+## `GET /lighthouse/beacon/health`
+
+Provides information about the sync status and execution layer health of each connected beacon node.
+For more inormation about how to interpret the beacon node health, see [Fallback Health](./redundancy.md#fallback-health).
+
+### HTTP Specification
+
+| Property          | Specification                              |
+|-------------------|--------------------------------------------|
+| Path              | `/lighthouse/beacon/health`                |
+| Method            | GET                                        |
+| Required Headers  | [`Authorization`](./api-vc-auth-header.md) |
+| Typical Responses | 200, 400                                   |
+
+### Example Response Body
+
+```json
+{
+    "data": {
+        "beacon_nodes": [
+            {
+                "index": 0,
```

Comment:
What does this `index` refer to? 

Says we have `--beacon-nodes http://localhost:5052,http://localhost:5053`. My understanding is `user_index` is the order of the endpoint set in this flag; so for endpoint 5052, `user_index=0` and for endpoint 5053, `user_index=1` .

Say endpoint 5052 is offline/out of sync, and endpoint 5053 is now returned first in the list in GET `/lighthouse/beacon/health`. Should the `index` for endpoint 5053 be 0 or 1 in this case? I may be misunderstood this field. I thought it represents the "current index/current rank" of the beacon node list. But I am seeing `index=1` for endpoint 5053 that's now returned first in the list.

Reply:
Thanks for the explanation! All good, I was having a misunderstanding that the `index` was referring to the current rank.